### DISCUSSION TITLE
2013-3 - 泛函分析基础：学习指导和习题详解

### DISCUSSION THREAD

**Discussion by giscus** [duskmoon314/THU_BFA#6] [Announcements]
# 2013-3 - 泛函分析基础：学习指导和习题详解

https://duskmoon314.github.io/THU_BFA/ch6/2013-3.html

**Comment by Timothy-Liuxf** [comment]
1. 用闭图像定理证明 $T \in B(X)$；$\lVert T \rVert = 1$ 易证（将 $t^2$ 放大成 1 得到 $\leq$；取 $x(t) = 1$ 得到 $\grq$）

**Reply by Timothy-Liuxf** [reply]
$\lVert T \rVert = 1$ 易证：
将 $t^2$ 放大成 $1$ 得到 $\lVert T \rVert \leq 1$
取 $x(t) = 1$ 得到 $\lVert T \rVert \geq 1$
